crates/pop-cli/src/commands/test/on_runtime_upgrade.rs

```diff
@@ -361,7 +367,7 @@ mod tests {
 				0, // live
 				None,
 			)
-			.expect_input("Enter the live chain of your node:", DEFAULT_LIVE_NODE_URL.to_string())
+			.expect_input("Enter the live chain of your node:", "ws://localhost:9944".to_string())
```

Comment:
I'd prefer to use a constant

Reply:
https://github.com/r0gue-io/pop-cli/pull/582/commits/cc619af503c84a725f7bd79e54bf345c70c610aa